docs/curl.1

```diff
@@ -150,6 +150,9 @@ version. (Added in 7.33.0)
 .IP "--http2"
 (HTTP) Tells curl to issue its requests using HTTP 2. This requires that the
 underlying libcurl was built to support it. (Added in 7.33.0)
+.IP "--http2Clean"
+(HTTP) Tells curl to issue its requests using HTTP 2 over clean TCP without HTTP/1.1 Upgrade. This requires that the
+underlying libcurl was built to support it. (Added in 7.33.0)
```

Comment:
Change 7.33 to target. I don't think this will be added until 7.49. 7.48 is a week away. Also wrap at 79